Podfile

```diff
@@ -31,6 +31,7 @@ end
 target 'Signal' do
   shared_pods
   pod 'ATAppUpdater', :inhibit_warnings => true
+  pod 'SSZipArchive', :inhibit_warnings => true
```

Comment:
There's a bunch of zip-for-iOS pods, but this one seems active, well-designed and it supports AES.